### DISCUSSION TITLE
Which SQL packages are supported?

### DISCUSSION THREAD

**Discussion by guettli** [stephenafamo/bob#160] [Q&A] [answered]
There are several ways to access sqlite via Go.

I am looking at that page: https://bob.stephenafamo.com/docs/code-generation/sqlite

Now I am unsure: Which integration to sqlite gets used.

Up to now I use https://pkg.go.dev/modernc.org/sqlite, but I can switch if this makes sense.

**Comment by stephenafamo** [comment, marked as answer]
It doesn't matter to Bob 🤷🏾 . The generated code requires a `bob.Executor` which can be created from any driver. (or even a mock for tests)

During generation, it uses [modernc.org/sqlite](https://pkg.go.dev/modernc.org/sqlite) to connect to the DB and read the structure, but this should be seen as an implementation detail.
